lms/templates/courseware/progress.html

```diff
@@ -59,15 +59,25 @@ <h1 class="progress-certificates-title">${_("Course Progress for Student '{usern
                     <% post_url = reverse('generate_user_cert', args=[unicode(course.id)]) %>
                     <div class="msg-content">
                         <h2 class="title">${_("Your certificate is available")}</h2>
+                        %if show_cert_web_view:
+                        <p class="copy">${_("You can now view your certificate.")}</p>
+                        %else:
```

Comment:
Does "You can now view your certificate" replace "Your certificate is available" if cert web view is enabled? If yes, I recommend making it even simpler and matching the "View Certificate" button text we use elsewhere.